docs/UserGuide.md

```diff
@@ -330,65 +331,132 @@ Example:
 
-### Saving the data
+To make the best out of your business, we have integrated statistical commands for you to filter, sort and find out which customers are your favorite.
 
-ArtBuddy data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+### List all the customers: `list`
+Lists all the customers
 
-### Editing the data file
+Format: `list`
 
-ArtBuddy data are saved as a JSON file `[JAR file location]/data/artbuddy.json`. Advanced users are welcome to update data directly by editing that data file.
+### Find a customer: `find`
+Finds all the customers who satisfy keyword matching in the name and tag filters. The tag filter more specifically will include customers who contain all tags under `-all` and at least one of the tags under `-any`.
+
+Format: `find [k/KEYWORDS]... -all [t/TAGS]... -any [t/TAGS]...`
+* The keywords, all, and any filters are all optional and can be omitted, but at least one should exist.
+
+Examples:
+* `find k/Kevin` Finds customers who have name Kevin.
+* `find -all t/friend t/colleague` Finds customers who are tagged both `friend` and `colleague`.
+* `find -any t/friend t/colleague` Finds customers who are either tagged `friend` or `colleague`.
+
+### Sorting the customer list: `sortcus`
+
+Sorts the displayed customer list by one of the following options:
+
+Prefix:
+- `n` (name)
+- `d` (latest commission date)
+- `c` (commission count)
+- `r` (revenue)
+- `a` (active commissions count)
+
+Suffix: `+` (increasing) or `-` (decreasing)
+
+Format: `sortcus PREFIX/SUFFIX`
+
+Examples:
+* `sortcus n/+` Sorts the customer list from A to Z.
+
+### List the commissions: `listcom`
+Lists all the commissions made by selected customer.
+
+Format: `listcom`
+
+### Viewing all commissions: `allcom`
+
+Displays all commissions across all customers in ArtBuddy.
+
+Format: `allcom`
+
+After running `allcom`, you should be switched to the commissions tab to view all commissions.
+To view the commissions for a specific customer, return to the customer list ([`opencus`](#opencus)) and select the customer from the list ([`opencus INDEX`](#opencus)).
+
+### Find a commission: `findcom`
+Finds all the commissions in the list which satisfy keyword matching in the title and tag filters. The tag filter more specifically will include commissions which contain all tags under `-all` and at least one of the tags under `-any`.
+
+Format: `find [k/KEYWORDS]... -all [t/TAGS]... -any [t/TAGS]...`
+* The keywords, all, and any filters are all optional and can be omitted, but at least one should exist.
 
+Examples:
+* `find k/Kevin` Finds customers who have name Kevin.
+* `find -all t/friend t/colleague` Finds customers who are tagged both `friend` and `colleague`.
+* `find -any t/friend t/colleague` Finds customers who are either tagged `friend` or `colleague`.
+
+## Miscellaneous
+ 
+### Clearing all the customers: `clear`
+Clears every customer in the ArtBuddy
```

Comment:
```suggestion
Clears all your customers from ArtBuddy, including their commissions and iterations.
```

Technically, because of how we store our data, commissions and iterations would also be cleared as well :P